contributing/code/bc.rst

```diff
@@ -253,6 +257,10 @@ Change return type                                  Yes
 **Static Methods**
 Turn non static into static                         No
 Turn static into non static                         No
+**Constants**
+Add constant                                        Yes
+Remove constant                                     No
+Change value of a constant                          Yes [5]_
```

Comment:
I agree with Bernhard.  You're right of course too Wouter, but probably constant value changes should be avoided in general and definitely covered in an UPGRADE file